kyo-core/shared/src/main/scala/kyo/internal/Trace.scala

```diff
@@ -10,10 +10,10 @@ object Trace:
     private val maxSnippetLines = 3
 
     extension (t: Trace)
-        def show: String       = t
-        def position: Position = Position(t.takeWhile(_ != '\n').drop(1))
-        def method: String     = t.drop(position.show.length + 2).takeWhile(isIdentifierPart)
-        def snippet: String    = t.drop(position.show.length + 2)
+        def show: String       = s"Trace(${position},method=${method},snippet=${snippet})"
```

Comment:
Do we need the `Trace` prefix?

Reply:
I put it to avoid having to label the values in `Suspend.toString`.